docs/tables/aws_kinesis_consumer.md

```diff
@@ -27,7 +27,7 @@ select
 from
   aws_kinesis_consumer
 where
-  consumer_arn = 'arn:aws:kinesis:us-east-1:986325076436:stream/turbot-data-stream/consumer/turbot-consumer:1616584220';
+  consumer_arn = 'arn:aws:kinesis:us-east-1:9863250123456:stream/my-data-stream/consumer/my-consumer:1616584220';
```

Comment:
Pls make sure the critical ID is not broken the length in sake of fake IDs 986325076436 > 9863250123456 has got 1 extra number

Reply:
resolved